extensions/common/json-ld/src/main/java/org/eclipse/edc/jsonld/transformer/JsonLdFunctions.java

```diff
@@ -0,0 +1,111 @@
+/*
+ *  Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.jsonld.transformer;
+
+import jakarta.json.JsonArray;
+import jakarta.json.JsonObject;
+import jakarta.json.JsonString;
+import jakarta.json.JsonValue;
+import org.eclipse.edc.transform.spi.TransformerContext;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.eclipse.edc.jsonld.transformer.JsonLdKeywords.ID;
+
+/**
+ * Contains methods for processing JSON-LD objects.
+ */
+public class JsonLdFunctions {
```

Comment:
same as `TransformerUtil`, the methods here are only used by transformers, could be moved in the abstract class